assets/get-started-sidebar.js

```diff
@@ -56,6 +62,17 @@ function showSidebar(osClass) {
     .show();
 }
 
+function showSidebar(cloudPartner, menuItem) {
```

Comment:
Can you change `cloudPartner` to be a more general argument name since both the cloud and local sidebars use this function? Also, it looks like the `showSidebar` function appears twice in the file. You should keep the one that has the "Hide all..." comment.